main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
-    ```
- 2. ```sh secondary style2
-    # Terminal 2: contract interaction
-    ```
- 3. ```sh secondary style3
-    # Terminal 3: web user interface
-    ```
+# Installing Curl
+Begin by installing the `curl` utility, if it's not already installed. If using Ubuntu you'll first need to run the command below to prevent an error when installing `curl`.
+`sudo apt-get update --fix-missing`
 
-::: tip Watch: Prepare Your Agoric Environment (November 2020)
-This presentation includes starting a project, but note an outdated detail:
+![006](./assets/006.png)
 
- - In the REPL `x~.go()` tildot support has been postponed; use `E(x).go()` instead.
+Install the `curl` utility.
+`sudo apt install curl`
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/w0By22jYhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-:::
+![007](./assets/007.png)
 
-## Initialize the Default Dapp Template
 
-The following section will explain how to initialize the default Dapp template, and install the Agoric SDK
-into the Dapp template.
+# Installing NVM and Node v18.16.0
+At this point the Node Version Manager (NVM) utility will be installed. NVM makes it easy to select the specific version of Node that will be required (v18.16.0).
+`curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash`
 
-Use the [Agoric CLI](../agoric-cli/) to fetch from a Dapp template
-and put it in a directory _not located in your `agoric-sdk` clone_. We named the directory "demo", but you can name the folder whatever you like.
+![008](./assets/008.png)
 
-```sh
-# Terminal 1
-# Don't use your agoric-sdk as the parent of the demo directory.
-cd $HOME
-agoric init --dapp-template dapp-card-store demo # use `agoric init --dapp-template dapp-card-store $DIRNAME` with any name you like
-cd demo
-agoric install community-dev # will take a minute to install all dependencies
-```
+Next, run the command:
+`source ~/.bashrc`
 
-Learn more about the [available Dapp templates](../dapps/dapp-templates.md).
+![009](./assets/009.png)
 
-## Start Agoric Solo Client and Simulated Blockchain
+Finally, install NVM with the command:
+`nvm install v18.16.0`
 
-```sh
-# Terminal 1
-agoric start --verbose --reset # `agoric start --reset` to start over
-```
+![010](./assets/010.png)
+
+
+# Installing Yarn
+Run the `corepack enable` command.
+`corepack enable`
 
-Leave this process and its logs running in its own terminal window.
+![023](./assets/023.png)
 
-## Open the Agoric Wallet and REPL
+Now run the `yarn –version` command
+`yarn –version`
 
-```sh secondary style2
-# Terminal 2
-cd demo
-agoric open --repl
+![049](./assets/049.png)
+
+
+# Installing Docker
+Now you'll install Docker using the two commands below. This first command will add the Docker GPG keys to your system, then add the repository to Apt for installation.
+```
```

Comment:
to get a bit of syntax highlighting:

````suggestion
```sh
````

Reply:
done